spec/controllers/system/admin/courses_controller_spec.rb

```diff
@@ -7,14 +7,10 @@
     let(:admin) { create(:administrator) }
     let(:instance_admin) do
       user = create(:user)
-      create(:instance_user, instance: instance, user: user, role: :administrator)
-      user
-    end
-    let(:normal_user) do
-      user = create(:user)
-      create(:instance_user, instance: instance, user: user, role: :normal)
+      user.instance_users.first.update_attributes(role: :administrator)
```

Comment:
haha, it's okay, these kinds of PRs it's easy to lose track...

i'll have to fetch and review again with whitespace changes removed when you push